test/internal/options.go

```diff
@@ -172,16 +194,40 @@ func (t *T) Prepare(ctx context.Context, k8sClient client.Client) error {
 		Expect(k8sClient.Create(ctx, profile)).To(Succeed())
 	}
 
+	if o.containerProfile != nil {
+		// Clone the placeholder profile
+		placeholder := &nnfv1alpha1.NnfContainerProfile{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "example-success",
+				Namespace: "nnf-system",
+			},
+		}
+
+		Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(placeholder), placeholder)).To(Succeed())
+
+		profile := &nnfv1alpha1.NnfContainerProfile{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      o.containerProfile.name,
+				Namespace: "nnf-system",
+			},
+		}
+
+		placeholder.Data.DeepCopyInto(&profile.Data)
+
+		Expect(k8sClient.Create(ctx, profile)).To(Succeed())
+	}
+
 	if o.cleanupPersistent != nil {
 		// Nothing to do in Prepare()
 	}
 
 	if o.persistentLustre != nil {
 		// Create a persistent lustre instance all the way to pre-run
 		name := o.persistentLustre.name
+		capacity := o.persistentLustre.capacity
 
 		o.persistentLustre.create = MakeTest(name+"-create",
-			fmt.Sprintf("#DW create_persistent type=lustre name=%s capacity=1TB", name))
+			fmt.Sprintf("#DW create_persistent type=lustre name=%s capacity=%s", name, capacity))
```

Comment:
I don't think there's any need to customize this with a test parameter. Just make it smaller 100GB

Reply:
Done